lib/ansible/modules/cloud/ovirt/ovirt_affinity_label_info.py

```diff
@@ -45,40 +47,44 @@
     host:
       description:
         - "Name of the host, which affinity labels should be listed."
-extends_documentation_fragment: ovirt_facts
+extends_documentation_fragment: ovirt_info
 '''
 
 EXAMPLES = '''
 # Examples don't contain auth parameter for simplicity,
 # look at ovirt_auth module to see how to reuse authentication:
 
-# Gather facts about all affinity labels, which names start with C(label):
-- ovirt_affinity_label_facts:
+# Gather information about all affinity labels, which names start with C(label):
+- ovirt_affinity_label_info:
     name: label*
+  register: result
 - debug:
-    var: affinity_labels
+    var: result.affinity_labels
```

Comment:
These examples were wrong before.
```suggestion
    msg: "{{ result.ovirt_affinity_labels }}"
```